probe_ble_specification.rst

```diff
@@ -0,0 +1,290 @@
+*************************
+Predictive Probe BLE Spec
+*************************
+
+:Authors: Jesse Johnston, Jason Machacek
+:status: DRAFT
+:version: 0.1h
+:date: March 29, 2022
```

Comment:
I think we should use Github to tag releases of this document.  I'd suggest removing Authors, version, and date.  Maybe leave status as Draft for now